### DISCUSSION TITLE
conditional bind

### DISCUSSION THREAD

**Discussion by ModProg** [hyprwm/Hyprland#5061] [Q&A] [answered]
I want to have a keybinding that  in general triggers an action directly, but when a specific application is focused, it will forward it instead.

Basically I'd like a way to add a `this app is (not)active` condition to the keybind.

**Comment by vaxerski** [comment]
you can do it with scripts

**Reply by ModProg** [reply]
Is there a command to send a key bind to the target application with hyprland?

**Reply by vaxerski** [reply, marked as answer]
no, but you can listen to the focused window and execute `hyprctl keyword bind` and `unbind` accordingly.
